release-notes-3.6.0.md

```diff
@@ -1,8 +1,8 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
+378627: Use worker-framework-testing
```

Comment:
I probably wouldn't call out 378627, I don't think consumers of this repo will care, it's really an internal detail.

I see in the commits the following change (I've paraphrased here) that maybe should be called out:

```
359648 Upgraded version of worker-framework, which eliminates the base64 encoding of `taskData` to reduce message size
```

@dermot-hardy Would you consider this change a 'New Feature' (so repos that consume it should be on a minor release)?

Or should we change this repo to a patch release and just indicate there were dependency updates? Like this:

```
#### New Features
- None

#### Bug Fixes
- Dependency updates only in this release.

#### Known Issues
- None
```

Reply:
Yes it is a new feature (inherited from the underlying framework).  Workers built on this new version of the Batch Worker Framework would be able to accept the new format message.  It is not a breaking change because they still accept the original format message also.  It is not a patch because it doesn't fix anything - it is a new feature that may be used but doesn't have to be.

Reply:
Looking at the underlying [Worker Framework v5.1.0 release notes](https://github.com/WorkerFramework/worker-framework/releases/tag/v5.1.0) I see that it's been correctly listed as a new feature, but the way that it is listed doesn't do it justice.  It says "Eliminated base64 encoding of taskData to reduce message size".  That almost makes it sound like a bug fix.  It should have been written up as a feature - something like: "Added support for a new leaner message format", and then in the paragraph below it could go on to explain that the `taskData` field no longer has to be base64-encoded.  Here are some better examples of writing up features:
- https://github.com/WorkerFramework/worker-framework/releases/tag/v1.6.0
- https://github.com/CAFDataProcessing/worker-document/releases/tag/v3.0.0
- https://github.com/CAFapi/opensuse-java8-images/releases/tag/v1.1.0